### ISSUE TITLE
Run tests?

### ISSUE THREAD

**Issue by dstroot** [storybookjs/design-system#70]
In package.json the tests reference react-scripts:

```json
    "test": "cross-env CI=1 react-scripts test --env=jsdom",
    "test:watch": "react-scripts test --env=jsdom"
```

But react-scripts are not installed.  How are y'all runnin yo tests?

**Comment by kylesuss** [comment]
@dstroot Nice catch. I suspect those scripts were ported over from another project. We don't have any tests right now, so I am just going to remove those until we do.

**Comment by kylesuss** [comment]
:rocket: Issue was released in v1.0.0 :rocket:
